datasafe-business-tests/datasafe-business-tests-random-actions/src/test/java/de/adorsys/datasafe/business/impl/e2e/randomactions/RandomActionsOnSimpleDatasafeAdapterTest.java

```diff
@@ -0,0 +1,225 @@
+package de.adorsys.datasafe.business.impl.e2e.randomactions;
+
```

Comment:
These Tests belong in the package datasafe-simple-adapter rather than in business.

Reply:
This tests are related to random action tests which do test business functionality, I don't see any sense to move two same tests in different packages

Reply:
For me it looks like a simpleDatasafeAdapter test. Further SimpleDatasafeAdapterImpl is build and depends on business, this is an ugly cycle.